src/pages/steps/LocationStep.tsx

```diff
@@ -464,7 +504,12 @@ const LocationStep = ({
               {scope === OfferTypes.EVENTS && OnlineToggle}
               <Inline spacing={5}>
                 {showRecentLocations && hasRecentLocations && (
-                  <RecentLocations flex={1} onFieldChange={onFieldChange} />
+                  <LocationSuggestions
+                    flex={1}
+                    locations={recentLocations}
+                    title={'create.location.recent_locations.title'}
```

Comment:
I would immediately pass the translation here:
```suggestion
                    title={t('create.location.recent_locations.title')}
```